shared/src/legalStructure.ts

```diff
@@ -132,6 +132,22 @@ export const LegalStructures: LegalStructure[] = [
       "dbaName",
```

Comment:
[dust] If we have a dedicated `foreign-nonprofit` legal type, we might be able to drop `dbaName` from this type.

Reply:
I don't think we *should* have `foreign-nonprofit` though so I think this needs to stay